### PR TITLE
[WIP] Trying to see if Travis can catch this sort of thing

### PR DESCRIPTION
**Pull request by Dref360**
### PR Overview

- [ ] This PR requires new unit tests [y/n] (make sure tests are included)
- [ ] This PR requires to update the documentation [y/n] (make sure the docs are up-to-date)
- [ ] This PR is backwards compatible [y/n]
- [ ] This PR changes the current API [y/n] (all API changes need to be approved by fchollet)

### CODE REVIEW

**Comment by Dref360** [comment]
@rragundez the output is kinda hard to parse :S if __init__ fails, everything fails.
We could catch the ImportError.

See: https://travis-ci.org/keras-team/keras-preprocessing/jobs/486494331

**Comment by fchollet** [comment]
One thing we should imperatively add:

- blank image, `python setup.py install` then `pip install keras`, then run a Keras integration test that involves preprocessing 
- blank image, `python setup.py install` then `pip install tensorflow`, then run a tf.keras integration test that involves preprocessing (the same as above but with different imports)
- may meed to be replicated for Keras at HEAD and for the tf-nightly build.
